frontend/src/database/types/internal/FindResults.tsx

```diff
@@ -0,0 +1,3 @@
+export interface FindResults {
+  docs: any[];
```

Comment:
Could this be changed to be more specific (easily)?

Reply:
Maybe using Generics?

Reply:
I already tried to get generics in it, but then it gets  incompatible with the types that pouchdb expects. Couldn't get it working with what I tried.